src/content/docs/setup/analytics/instrumentation.mdx

```diff
@@ -1,58 +1,205 @@
 ---
 title: Instrumentation
-description: Learn how to instrument your Adobe Commerce on Edge Delivery Services storefront project.
+description: Learn how to instrument your Adobe Commerce on Edge Delivery Services storefront project to collect user interaction events for analytics and personalization.
 ---
 
 import OptionsTable from '@components/OptionsTable.astro';
+import Aside from '@components/Aside.astro';
+import CodeExample from '@components/CodeExample.astro';
+import { Steps } from '@astrojs/starlight/components';
+import Link from '@components/Link.astro';
 
-For Live Search and Product Recommendation APIs to work correctly, you must collect and send user interaction events to Commerce.
+## Overview
 
-User interaction events power Adobe Sensei features, like intelligent merchandising for Live Search and to personalize product recommendations units. These events also populate performance dashboards for [Live Search](https://experienceleague.adobe.com/en/docs/commerce/live-search/live-search-admin/performance).
+Instrumentation is the process of adding code to your storefront to collect data about user interactions and behaviors. This data powers critical Adobe Commerce features and provides valuable insights into your customers' shopping journey.
+
+### Why Instrumentation Matters
+
+User interaction events collected through instrumentation enable:
+
+- **Adobe Sensei features**: Intelligent merchandising and search result optimization in Live Search
+- **Product recommendations**: Personalized product suggestions based on user behavior  
+- **Performance analytics**: Detailed dashboards showing search performance, conversion rates, and user engagement
+- **Business intelligence**: Data-driven insights for inventory management and marketing strategies
+
+<Aside type="caution" title="Required for Core Features">
+For Live Search and Product Recommendations to function correctly, you must collect and send user interaction events to Adobe Commerce. Without proper instrumentation, these features will not work as expected.
```

Comment:
just for the record, Live Search and Product Recommendations events are not sent to AEP, only snowplow (I just was confused as PR title is about AEP)